samples/compose-samples/src/androidTest/java/com/squareup/sample/compose/hellocompose/HelloComposeTest.kt

```diff
@@ -25,13 +26,19 @@ class HelloComposeTest {
     .around(IdlingDispatcherRule)
 
   @Test fun togglesBetweenStates() {
-    composeRule.onNodeWithText("Hello")
-      .assertIsDisplayed()
-      .performClick()
-    composeRule.onNodeWithText("Goodbye")
-      .assertIsDisplayed()
-      .performClick()
-    composeRule.onNodeWithText("Hello")
-      .assertIsDisplayed()
+    runBlocking {
+      composeRule.onNodeWithText("Hello")
+        .assertIsDisplayed()
+        .performClick()
+      composeRule.awaitIdle()
```

Comment:
@RBusarow - after the change in the runtime to process multiple actions we introduce a suspending yield - for the onTimeout select clause even if there is only 1 action to process. This means that with the main immediate dispatcher tied to the IdlingResource watcher we don't get immediate synchronous results to assert on.

We have to wait and then advance the time in order to stabilize these tests - I need to talk with you about how to do this in a better general way.